scipy/interpolate/src/_fitpackmodule.c

```diff
@@ -159,8 +159,8 @@ static char doc_bispev[] = " [z,ier] = _bispev(tx,ty,c,kx,ky,x,y,nux,nuy)";
 static PyObject *
 fitpack_bispev(PyObject *dummy, PyObject *args)
 {
-    F_INT nx, ny, kx, ky, mx, my, lwrk, *iwrk, kwrk, ier, lwa, nux, nuy;
-    npy_intp mxy;
+    F_INT nx, ny, kx, ky, lwrk, *iwrk, kwrk, ier, lwa, nux, nuy;
+    unsigned mx, my, mxy;
```

Comment:
The cast to `size_t` (not unsigned) should be made only where the multiplication is made, since these have to be of type `F_INT` for passing to Fortran.

Reply:
From the code logic there is no difference, and `unsigned` value wont be truncated, and the type int/unsigned would be implicitly cast.

Reply:
No, Fortran passes by reference.

Reply:
I didnt notice that there was `int to F_INT` change that has native pointer size, since I am working with scipy 1.4.1 and there is an old interface. Cause when i was talking 'truncated' I was talking about the value not the variable size.